packages/react-reconciler/src/__tests__/ReactIncrementalErrorHandling-test.js

```diff
@@ -792,14 +794,16 @@ describe('ReactIncrementalErrorHandling', () => {
     expect(ReactNoop.flush).toWarnDev(
       'Warning: React.createElement: type is invalid -- expected a string',
     );
-    expect(ReactNoop.getChildren()).toEqual([
+    expect(ReactNoop.getChildren()).toErrorWithStack([
```

Comment:
This is not an error. It’s comparing the actual children (because the error was “rendered” into the DOM by an error boundary).

Just change this to `toContain` and completely drop the part with the stack. It doesn’t matter because we have other tests verifying the stack works.